### PR TITLE
Do work in the wdl working directory

### PR DESCRIPTION
**Pull request by ttung**
### Description
This facilitates copying of data out of the runs if need be.

Depends on #501 

### Test plan
started a run.  It's not yet done.

### CODE REVIEW

File: .happy/terraform/modules/sfn_config/nextstrain.wdl [hunk 1]
```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
...
-    cd /ncov
-    git init
-    git fetch --depth 1 git://github.com/nextstrain/ncov.git
-    git checkout FETCH_HEAD
-    ncov_git_rev=$(git rev-parse HEAD)
+    mkdir -p ncov/my_profiles/aspen ncov/results
+    (cd ncov &&
+     git init &&
+     git fetch --depth 1 git://github.com/nextstrain/ncov.git &&
+     git checkout FETCH_HEAD
```
Comment (danrlu) [inline .happy/terraform/modules/sfn_config/nextstrain.wdl:75]: i've been wanting to ask: what does this do? to get the hash?

File: .happy/terraform/modules/sfn_config/nextstrain.wdl [hunk 2]
```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
...
     aligned_gisaid_metadata_s3_key=$(echo "${aligned_gisaid_location}" | jq -r .metadata_key)
 
     # fetch the gisaid dataset
-    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" - | zstdmt -d | xz -2 > /ncov/results/aligned_gisaid.fasta.xz
-    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" /ncov/data/metadata_gisaid.tsv
+    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_sequences_s3_key}" - | zstdmt -d | xz -2 > ncov/results/aligned_gisaid.fasta.xz
+    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" ncov/data/metadata_gisaid.tsv
 
-    snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/  --resources=mem_mb=312320 || aws s3 cp /ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
+    (cd ncov && snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/ --resources=mem_mb=312320) || aws s3 cp ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
```
Comment (danrlu) [inline .happy/terraform/modules/sfn_config/nextstrain.wdl:97]: would `cd ncov` here influence the following `aws s3 cp ncov/...` and in line 101, 119 or not?

Comment (ttung) [reply .happy/terraform/modules/sfn_config/nextstrain.wdl:97]: No, it's in a subshell, so it doesn't affect the rest.

File: .happy/terraform/modules/sfn_config/nextstrain.wdl (hunk 1 above)
Comment (ttung) [reply .happy/terraform/modules/sfn_config/nextstrain.wdl:75]: which line?

Comment (danrlu) [reply .happy/terraform/modules/sfn_config/nextstrain.wdl:75]: `git checkout FETCH_HEAD`

**Review by danrlu — APPROVED** [review]
I tested out this version of nextstrain-dbg.wdl too. Works beautifully!

File: .happy/terraform/modules/sfn_config/nextstrain.wdl (hunk 1 above)
Comment (ttung) [reply .happy/terraform/modules/sfn_config/nextstrain.wdl:75]: When one runs `git fetch`, it creates a temporary branch called "FETCH_HEAD" that represents the HEAD of what you just fetched.

**Comment by danrlu** [comment]
Checks are not moving. I'll open a new PR.

**Pull request closed without merging**